iree/tools/BUILD

```diff
@@ -111,6 +111,16 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "Translations",
+    hdrs = [
+        "init_translations.h",
+    ],
+    deps = [
+        "SPIRVTranslateRegistration",
```

Comment:
```suggestion
        ""@llvm-project//mlir:SPIRVTranslateRegistration"",
```

Reply:
Obviously... Thanks for catching this!

Reply:
Except I've got two many quotes in my suggestion :-D

Reply:
Also I didn't, the CI did :-P

Reply:
Yeah, and I used github to submit :see_no_evil: Anyway, only needed to rewrite the commit message.